ppcs/ppc0019-qt-string.md

```diff
@@ -70,6 +70,38 @@ will be equivalent to
 qq{A ${ \scalar(TEXT) } B};
 ```
 
+Escape sequences that are recognised in `qq` strings as standing in place for
+other literal characters (such as `\n` for linefeed or `\x00` for a NUL byte)
+are also recognised by `qt`.
+
+```perl
+print qt{This message ends in a linefeed\n};
+
+print qt{As does this message\x0d\x0a};
+
+print qt{It's Christmas! \N{SNOWMAN} };
+```
+
+The `$` and `@` symbols are *not* considered special and will act literally,
+as they behave inside a `q` string.
+
+```perl
+print qt{This apple will cost you $1.\n};
+# will print a literal '$' symbol followed by the digit 1, it does not
+# interpolate the contents of the first regexp capture buffer
+```
+
+Likewise, the escape sequences `\L`, `\l`, `\U`, `\u`, `\Q` and `\E`, which in
+`qq` strings control the way that variables are interpolated, are also not
+considered special here.  It would be helpful to users to at least issue a
+warning in this case, to remind them of this fact.
+
+```
+say qt{This does not uppercase the variable \U{ $var }};
+
+Use of the \U escape sequence is not considered special by qt quoting at FILE line LINE.
```

Comment:
Why not reuse the existing message?
```
$ perl -wE '"" =~ tr/\U//'
Unrecognized escape \U passed through at -e line 1.
```

Reply:
Oh that's true; I forgot there was already precident there. Might as well reüse it yes.